cmd/create/cluster/cmd.go

```diff
@@ -1322,9 +1356,8 @@ func run(cmd *cobra.Command, _ []string) {
 				subnetIDs[i] = aws.ParseSubnet(subnet)
 			}
 		}
-
 		// Validate subnets in the case the user has provided them using the `args.subnets`
-		if subnetsProvided {
+		if subnetsProvided && !isHostedCluster {
 			err = ocm.ValidateSubnetsCount(multiAZ, privateLink, len(subnetIDs))
```

Comment:
Maybe this validation function can be modified to validate the number of subnet values based on the privacy values of the HC

Reply:
I'd rather have that in a separate PR. Since M1 is for the internal preview and we explicitly ask the users to create subnets, we can skip this.